.github/workflows/greetings.yaml

```diff
@@ -0,0 +1,16 @@
+name: Greetings
+
+on: [pull_request_target, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: "Hi there! Thanks for opening this issue. We appreciate your contribution to this open-source project. We aim to respond or assign your issue as soon as possible. In the meantime you can star the repo and read the contributing.md"
```

Comment:
Please replace the `contributing.md` part with `[contributing.md](https://github.com/singodiyashubham87/Draw-it-out/blob/main/CONTRIBUTING.md)`.

We need to add actual link instead of plain text.

Once you made changes, we're good to go 🚀

Reply:
sure Ill do it